### DISCUSSION TITLE
Hide visibility toggles for columns where enableHiding is false.

### DISCUSSION THREAD

**Discussion by benjaminw1030** [KevinVandy/material-react-table#549] [Q&A]
Hello,

By default, the dropdown for Show/Hide Columns displays a disabled toggle along with the column name when enableHiding is not set to true for that column. Is there a way to not display this toggle at all? We store our row action buttons in a separate column with no title and this causes a lone disabled toggle with no text to render in the dropdown.

See examples:
![Empty Label](https://github.com/KevinVandy/material-react-table/assets/59542595/1a2bbc01-eea6-4551-a2e2-677d6b8df256)
![No Title Column](https://github.com/KevinVandy/material-react-table/assets/59542595/0a99ce1c-e2a1-4b0c-9aea-1d2a90a9295d)

**Comment by joey-kwl** [comment]
> Is there a way to not display this toggle at all?

No, I don't think this is possible: https://github.com/KevinVandy/material-react-table/blob/main/packages/material-react-table/src/menus/MRT_ShowHideColumnsMenuItems.tsx#L130
